sdk/alert.ts

```diff
@@ -82,7 +82,7 @@ type AlertInput = {
 export class Alert {
   private readonly addressFilter?: BloomFilter;
 
-  private constructor(
+  constructor(
```

Comment:
lets keep this private. we want Alert.fromObject to be the only way to initialize an Alert object